pyprof/prof/memory.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2020, Aditya Agrawal.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .base import OperatorLayerBase
+from .tensor import Tensor
+
+def readMarker(d):
+    marker = eval(d.argMarker[0])
+    return marker['mod'], marker['op'], marker['args']
+
+class OneZero(OperatorLayerBase):
```

Comment:
Can you comment this class to give some indication what it is for?

Reply:
Done.